documentation/proc-pages/development/standards.md

```diff
@@ -20,16 +26,46 @@ real(dp) :: b
 
 ```
 
+all new models should have their own function
+
+--------------------
+
 ## Naming conventions
 
+!!! quote "The Zen of Python"
+
+    *“Explicit is better than implicit.”*
+
+    *"Readability counts."*
+
+### Functions
+
+Use a lowercase word or words. Separate words by underscores(`_`) to improve readability.
+
+### Switches
+
+Switches should start with the `i_` prefix in their name, be of integer type and should be indexed from 0.
+Switches should by their nature not override other switches.
```

Comment:
Could you clarify this? There are current cases (and legitimate future ones) where defining a switch as one value means that others are ignored or are invalid. Perhaps something could be stated about these scenarios being caught, so switches aren't tacitly ignored in this case, but instead should throw exceptions?